### ISSUE TITLE
Possible to connect debugger to observatory url?

### ISSUE THREAD

**Issue by MostHated** [flutter/flutter-intellij#3724]
Hey there,
    My current app uses go-flutter and as such I have to start the application using ```hover run```, but once it starts up it provides me with an observatory listening url which I can then take and pop into DevTools and everything works just fine. I tried adding that url to the dart remote debugger, but that didn't work. I didn't expect it to, but was hopeful. Is there a way to manually connect the debugger to that url? (Ex. http://127.0.0.1:50300/yLwAgKUj2w0=/ ). I saw in the newest update to the plugin that Desktop and Web were back in, but unfortunately, they don't see when I run my application.

Thanks,
-MH

**Comment by stevemessick** [comment]
I don't think it is possible to do what you ask today. I doubt we have any plans to make it happen, given that Dart DevTools is largely a replacement for Observatory UI. Personally, I think connecting to a running Dart VM would be handy in enterprise debugging scenarios. Also, connecting to a Dart analysis server running remotely would be valuable, too. @devoncarew do we have any plans for either?
